### PR TITLE
fix(text-metrics): height measurement

### PR DESCRIPTION
**Pull request by cbt1**
Changes how the height of a text is measured to a method based on the font-size pixel value. This will make it less prone to issue with fonts that have narrow style for the character `M`.

See example below using `Arial` for the difference when computing the bounding rectangle of a text. Where the red rectangle is the bounds.

**Before**
Screenshot 2019-04-24 at 13 58 47

**After**
Screenshot 2019-04-24 at 13 58 33

### CODE REVIEW

File: packages/picasso.js/src/web/text-manipulation/font-size-to-height.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const BASE = 24;
+const PAD = 4;
+const BUMP = 1e-12;
+const DEFAULT_HEIGHT = 16;
+const TEXT_REGEX = /^\s*([0-9]+|[0-9]+\.{1}[0-9]+)[px]{2}\s*$/i;
```
Comment (miralemd) [inline packages/picasso.js/src/web/text-manipulation/font-size-to-height.js]: simplify the regex to `/^\s*\d(\.\d+)?px\s*/`

File: packages/picasso.js/src/web/text-manipulation/font-size-to-height.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+function isValidFontSize(val) {
+  const type = typeof val;
```
Comment (miralemd) [inline packages/picasso.js/src/web/text-manipulation/font-size-to-height.js]: could write all of this as `return typeof value === 'string' ? TEST_REGEX.test(val) : false`

File: packages/picasso.js/src/web/text-manipulation/font-size-to-height.js (hunk 1 above)
Comment (cbt1) [reply packages/picasso.js/src/web/text-manipulation/font-size-to-height.js]: Doesn't work for multiple number, ex: 90px. But will fix n update.

Comment (cbt1) [reply packages/picasso.js/src/web/text-manipulation/font-size-to-height.js]: Also regarding case, I've tried to follow the browser implementation for SVG text element. Where in FF, chrome and safari, PX and px is allowed.

**Pull request merged**